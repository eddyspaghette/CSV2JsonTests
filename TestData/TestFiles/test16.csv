ッ,あ,が,ざ,℁,⏳
デ,う,こ,₢,Δ,⭐
ヅ,い,け,₠,Å,○
ミ,ぅ,げ,,ℬ,☝
ツ,,ぎ,せ,℟,▓
,ぇ,ご,₣,ψ,
ド,え,さ,℀,ↂ,🍕
