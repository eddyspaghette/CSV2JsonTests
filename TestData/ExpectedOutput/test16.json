[
 {
   "ッ": "デ",
   "あ": "う",
   "が": "こ",
   "ざ": "₢",
   "℁": "Δ",
   "⏳": "⭐"
 },
 {
   "ッ": "ヅ",
   "あ": "い",
   "が": "け",
   "ざ": "₠",
   "℁": "Å",
   "⏳": "○"
 },
 {
   "ッ": "ミ",
   "あ": "ぅ",
   "が": "げ",
   "ざ": "",
   "℁": "ℬ",
   "⏳": "☝"
 },
 {
   "ッ": "ツ",
   "あ": "",
   "が": "ぎ",
   "ざ": "せ",
   "℁": "℟",
   "⏳": "▓"
 },
 {
   "ッ": "",
   "あ": "ぇ",
   "が": "ご",
   "ざ": "₣",
   "℁": "ψ",
   "⏳": ""
 },
 {
   "ッ": "ド",
   "あ": "え",
   "が": "さ",
   "ざ": "℀",
   "℁": "ↂ",
   "⏳": "🍕"
 }
]